playwright/base.ts

```diff
@@ -0,0 +1,76 @@
+import { test as base, expect } from "@playwright/test";
+
+interface Log {
+    msg: any;
+    type: string;
+}
+
+// export the extended `test` object
+export const test = base.extend<{ page: void; failOnJSError: boolean; displayAsciiTable: boolean }>({
```

Comment:
Seems like overwriting the `page` type with void breaks the type checking for this module. If you remove the `page: void` and let typescript infer the type of `page` it works fine